common/lib/xmodule/xmodule/raw_module.py

```diff
@@ -22,7 +23,13 @@ class RawMixin(object):
 
     @classmethod
     def definition_from_xml(cls, xml_object, system):
-        return {'data': etree.tostring(xml_object, pretty_print=True, encoding='unicode')}, []
+        pre_tag_data = [etree.tostring(pre_tag_info) for pre_tag_info in xml_object.findall('pre')]
+        pre_tag_pattern = re.compile(r'<pre>[\s\S]*?</pre>')
```

Comment:
Can we separate the definition of the regex from its use? Maybe define it on the top of the class?